Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,250 @@
+                return InvalidNumberInput;
+            }
+
+            return weightInKg / heightInM / heightInM;
+        }
+
+        /// <summary>
+        /// Reads in the data of a person and processes it further.
+        /// </summary>
+        private static void ProcessPerson()
+        {
+            string fullName = PromptString("What is your full name? ");
+            int age = PromptInt("What is your age? ");
+            float weightInKg = PromptFloat("What is your weight in kg? ");
+            float heightInCm = PromptFloat("What is your height in cm? ");
+
+            Console.WriteLine($"{fullName} is {age} years old, his weight is {weightInKg} kg and his height is {heightInCm} cm.");
+            Console.WriteLine($"His BMI is {CalculateBmi(weightInKg, heightInCm / 100)}.");
+        }
+
+        /// <summary>
+        /// Prompts for an integer value. Converts the string input to an integer if possible.
+        /// </summary>
+        /// <param name="message">The message to print before the prompt.</param>
+        /// <param name="result">The parsing result. Is 0 if the parsing failed.</param>
+        /// <param name="input">The user input.</param>
+        /// <returns>Returns true if the parsing succeeded, false otherwise.</returns>
+        private static bool PromptTryParse(string message, out int result, out string input)
+        {
+            input = PromptInput(message);
+            return int.TryParse(input, out result);
+        }
+
+        /// <summary>
+        /// Prompts for a float value. Converts the string input to a float if possible.
+        /// </summary>
+        /// <param name="message">The message to print before the prompt.</param>
+        /// <param name="result">The parsing result. Is 0 if the parsing failed.</param>
+        /// <param name="input">The user input.</param>
+        /// <returns>Returns true if the parsing succeeded, false otherwise.</returns>
+        private static bool PromptTryParse(string message, out float result, out string input)
+        {
+            input = PromptInput(message);
+            return float.TryParse(input, NumberStyles.Float, CultureInfo.InvariantCulture, out result);
+        }
+
+        /// <summary>
+        /// Prints the message and reads in the user input.
+        /// </summary>
+        /// <param name="message">The message to print.</param>
+        /// <returns>The unparsed and unvalidated input.</returns>
+        private static string PromptInput(string message)
+        {
+            Console.WriteLine(message);
+            return Console.ReadLine();
+        }
+
+        /// <summary>
+        /// Returns a value that indicates the type of input fault./>
+        /// </summary>
+        /// <param name="input">The input the user wrote.</param>
+        /// <returns>An integer indicating the error.</returns>
+        private static int InvalidInput(string input)
+        {
+            if (string.IsNullOrEmpty(input))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4885761)